opennms-webapp-rest/src/main/java/org/opennms/web/rest/v1/EventRestService.java

```diff
@@ -67,6 +69,9 @@
     @Autowired
     private EventDao m_eventDao;
 
+    @Autowired
+    private EventIpcManager m_eventForwarder;
```

Comment:
Autowiring the EventForwarder fails because of multiple implementations.
Adding a Qualifier fixes this, but causes the tests to fail.